### DISCUSSION TITLE
A compressed file with the password 'changeme' has invaded the Github issue area.

### DISCUSSION THREAD

**Discussion by Moemu** [community/community#137322] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hi everyone, I found out that this guy has invaded my issue. At the same time, I discovered that his guy has left this message in any corner of Github. Can you help us determine if the application behind this link is harmful? Is there a way to permanently shut them up?  (They don't seem to be alone)

<img width="743" alt=" 2024-08-31 153547" src="https://github.com/user-attachments/assets/0221f8ac-1c2a-440d-a0be-6009a7d30bb1">

**Comment by joemar25** [comment, marked as answer]
Hey @Moemu ,

I saw your post about that suspicious link someone dropped in your GitHub issue. That’s definitely concerning, especially when it involves a compressed file with a password like "changeme." It’s a classic red flag.

First off, **don’t download or open that file**. It could easily contain malware, viruses, or other nasty stuff that you definitely don’t want on your system.

If you want to check whether that link is actually harmful, there are a few online tools you can use. For example, **VirusTotal** is great. You just paste the URL, and it’ll scan it using tons of antivirus engines and tell you if anything fishy is going on. There’s also **Google Safe Browsing**, **URLVoid**, and a few others that do similar things. They’ll let you know if the link has been reported as unsafe or flagged by security databases.

Here’s what I’d recommend:
1. **Report that comment and the user to GitHub**. Use their "Report abuse" feature. It’s important to get this stuff flagged.
2. **Block the user** if you have the permissions in your repo. This will stop them from posting more spam.
3. **Delete the comment** to protect others who might not realize it’s malicious.
4. **Check your repo** for other similar comments or spam—just to be safe.

It might also be a good idea to enable some moderation features in your repo, like approving the first comment from new users, to help catch this kind of thing in the future.

If you or your team need more help figuring this out, just let me know! Better safe than sorry with these things.

Stay safe, and hope that helps!
